wasm-rpc-stubgen/test-data/caller-no-dep-importstub/wit/caller.wit

```diff
@@ -5,7 +5,7 @@ interface api {
 }
 
 world caller {
-    import test:main-stub/stub-api;
+//!!    import test:main-stub/stub-api;
```

Comment:
Why is this commented out?

Reply:
discussed: it gets uncommented in the test, but now it required to be a valid wit before adding the test